### ISSUE TITLE
Unit Deployment Management

### ISSUE THREAD

**Issue by Masashi86** [WallRushGO/ASMGameJam2018#5]
How to deploy Units. Select unit in UI and deploy it, and it will be instantiated to specific position. NOTE: do we have resource management as own feature?

**Comment by WallRushGO** [comment]
Yes, resource management as its own feature please. Keep the tickets as small as possible.
